tests/php/Security/MemberAuthenticator/CookieAuthenticationHandlerTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace SilverStripe\Security\Tests\MemberAuthenticator;
+
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\MemberAuthenticator\CookieAuthenticationHandler;
+
+class CookieAuthenticationHandlerTest extends SapphireTest
+{
+
+    protected $usesDatabase = true;
+
+    public function testLoginOnlySetsTokenWithPersistent()
+    {
+        $member = new Member(['Email' => 'test@example.com']);
+        $member->write();
+
+        $auth = new CookieAuthenticationHandler();
+        $auth->setTokenCookieName('testTokenCookie');
+
+        $headers = $this->runWithHeaderIsolation(function () use ($auth, $member) {
+            $auth->logIn($member, $persistent = true);
+        });
+        $cookieHeader = $this->getCookieHeader($headers);
+        $this->assertNotContains("Secure", $cookieHeader);
+
+        $auth->setTokenCookieSecure(true);
```

Comment:
This method doesn't seem to exist anywhere

Reply:
PR crossover from https://github.com/silverstripe/silverstripe-framework/pull/8261, I'll remove the test here - it's the same area, but not really related to the PR at hand, my mistake.